add stable diffusion notebook (#642)
openvinotoolkit/openvino_notebooks: notebooks/224-stable-diffusion-text-to-image/README.md

> # Text-to-image generation with Stable Diffusion

Reviewer: ```suggestion
# Text-to-Image Generation with Stable Diffusion
```

> Text-to-image generation is task of AI, which aim is create image from given text description.

Reviewer: ```suggestion
Text-to-image generation using an AI method called "diffusion" enables us to create images from a text description as input.
```

> In this demo we use the **[Stable Diffusion](https://huggingface.co/CompVis/stable-diffusion)** model for image generation with OpenVINO.

Reviewer: ```suggestion
In this notebook we demonstrate how to use a **[Stable Diffusion](https://huggingface.co/CompVis/stable-diffusion)** model for image generation with OpenVINO.
```

> The complete pipeline of this demo's notebook is shown below.

Reviewer: ```suggestion
The complete pipeline of this demo is shown below.
```

![image2](https://raw.githubusercontent.com/patrickvonplaten/scientific_images/master/stable_diffusion.png)

> This is a demonstration in which the user can type the text and the pipeline will generate an image that reflects the context of the given text, step by step iteratively denoise latent image representation while being conditioned on the text embeddings provided by text encoder. 

Reviewer: ```suggestion
This is a demonstration in which the user can type a text description and the pipeline will generate an image that reflects the context of the input text. Step-by-step the diffusion process will iteratively denoise latent image representation while being conditioned on the text embeddings provided by the text encoder. 
```

The following image shows an example of the input sequence and corresponding predicted image.

> ![image](data/Corgi.png)

Reviewer: ```suggestion
![image](https://user-images.githubusercontent.com/15709723/200945747-1c584e5c-b3f2-4e43-b1c1-e35fd6edc2c3.png)
```
Please remove the corgi.png image from the repo to save space

> # Text-to-image generation with Stable Diffusion and OpenVINO

Reviewer 2: "... and OpenVINO" in the and of the title looks excessive considering that this is all about OpenVINO. You can add "and OpenVINO native API" if you want to highlight implementation details or remove it at all.

Author: removed

Text-to-image generation is task of AI, which aim is create image from given text description.
In this demo we use the **[Stable Diffusion](https://huggingface.co/CompVis/stable-diffusion)** model for image generation with OpenVINO.

The complete pipeline of this demo's notebook is shown below.

![image2](https://raw.githubusercontent.com/patrickvonplaten/scientific_images/master/stable_diffusion.png)

> This is a demonstration in which the user can type the text and the pipeline will generate image which correspond to given text, step by step iteratively denoise latent image representation while being conditioned on the text embeddings provided by text encoder. 

Reviewer 2: "will generate image which correspond to given text" -> "will generate an image which corresponds to given text"

Author: fixed

> This is a demonstration in which the user can type the text and the pipeline will generate image which correspond to given text, step by step iteratively denoise latent image representation while being conditioned on the text embeddings provided by text encoder. 

Reviewer 2: I would rephrase "which correspond to given text" to "that reflects the context of the given text"

Author: rephrased

> The following image show an example of the input sequence and corresponding predicted image.

Reviewer 2: "show" -> "shows"

![image](data/Corgi.png)

## Notebook Contents

> This notebook demostrates how convert and run stable diffusion using OpenVINO.

Reviewer 2: "demostrates" -> demonstrates

> This notebook demostrates how convert and run stable diffusion using OpenVINO.

Reviewer 2: "how convert" -> "how to convert"

Author: fixed, thanks

> Notebook contains following steps:

Reviewer 2: "following" -> "the following"

> 1. Convert model to ONNX format

Reviewer 2: 1. -> "1. Convert PyTorch models to ONNX format."
2. -> "2. Convert ONNX models to OpenVINO IR format using Model Optimizer tool."
3. -> "3. Run Stable Diffusion pipeline with OpenVINO."

Author: updated

2. Convert model to IR using OpenVINO Model Optimizer
3. Run Stable Diffusion pipeline with OpenVINO models

## Model Access

> You need to accept the model license before downloading or using the weights. In this demo we'll use model version `v1-4`, so you'll need to  visit [its card](https://huggingface.co/CompVis/stable-diffusion-v1-4), read the license and tick the checkbox if you agree. 

Reviewer 2: I think it is better to use the full name of the version: "`stable-diffusion-v1-4`"

Reviewer 2: The same comments/fixed can be transferred to the beginning of the notebook.

Author: applied
